.github/workflows/capture-sdk.build.docs.yml

```diff
@@ -1,13 +1,11 @@
 name: Builds docs for Capture SDK
 
 on:
-  push:
-    paths:
-      - 'CaptureSDK/**'
   workflow_dispatch:
     tags-ignore:
```

Comment:
[clarification]Why for most of the checks we removed this and for this not? Mostly `docs check` has this.

Reply:
Because it's just a build.docs. It doesn't need to run on every push. 
But publish docs still have push trigger

Reply:
Thank you for the clarification!